web/controllers/registration_controller.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Yggdrasil.RegistrationController do
+  use Yggdrasil.Web, :controller
+
+  import Ecto.Changeset, only: [put_change: 3]
+  import Comeonin.Bcrypt, only: [hashpwsalt: 1]
+
+  alias Yggdrasil.User
+
+  # if logged in, go straight to client
+  plug Yggdrasil.Plug.SkipIfAuthenticated
+  plug :scrub_params, "user" when action in [:create]
+
+  def new(conn, _params) do
+    changeset = User.changeset(%User{})
```

Comment:
Why do you have to use the `changeset()` function? You can't render the `%User{}` struct directly?